lib/sylius/normalizer/product-normalizer.ts

```diff
@@ -12,7 +12,10 @@ export const normalizeProduct = (product: SyliusProduct): Product => ({
     title: product.name,
     description: product.shortDescription
   },
-  variants: product.variants.map((variant) => normalizeProductVariant(variant)),
+  // dont need varaints when serializing fort cart
```

Comment:
typos (on peut laisser si on s'en fout mais je remonte au cas où)

variants not needed for cart